### ISSUE TITLE
安装openelb后，再安装ingress-nginx报错？

### ISSUE THREAD

**Issue by shiwu32** [openelb/openelb#238]
实验环境：kubekey在centos7安装kubernetes

#1 安装openELB控制器
kubectl apply -f https://raw.githubusercontent.com/kubesphere/porter/master/deploy/porter.yaml
#2 vim eip.yaml
root@master01:~# cat eip.yaml 
apiVersion: network.kubesphere.io/v1alpha2
kind: Eip
metadata:
  name: eip-pool
spec:
  address: 172.26.196.91-172.26.196.92 
  protocol: layer2
  interface: eth0 ## 网卡的名称
  disable: false

kubectl apply -f eip.yaml

#3 再安装ingress-nginx报错
kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.1.0/deploy/static/provider/cloud/deploy.yaml
Error from server (InternalError): error when creating "https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.1.0/deploy/static/provider/cloud/deploy.yaml": Internal error occurred: failed calling webhook "mutating.eip.network.kubesphere.io": the server could not find the requested resource
Error from server (InternalError): error when creating "https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.1.0/deploy/static/provider/cloud/deploy.yaml": Internal error occurred: failed calling webhook "mutating.eip.network.kubesphere.io": the server could not find the requested resource

注：我先安装ingress-nginx再安装openelb
在edit svc 
ingresss-nginx/ingress-nginx-controller
# 需要对服务进行配置,通过以下两条信息来识别为哪些svc暴露服务的
annotations:
  lb.kubesphere.io/v1alpha1: porter
  protocol.porter.kubesphere.io/v1alpha1: layer2
---
这里也会报跟安装的"mutating.eip.network.kubesphere.io"类似错误

**Comment by chaunceyjiang** [comment]
Maybe your `porter-manager` doesn't start normally. 
Run the following command to check whether the status of porter-manager is `READY: 1/1` and `STATUS: Running`. If yes, PorterLB has been installed successfully.
```shell
kubectl get po -n porter-system
```

**Comment by chaunceyjiang** [comment]
This may be the reason.
https://github.com/openelb/openelb/blob/master/deploy/porter.yaml#L1146-L1147

**Comment by FeynmanZhou** [comment]
> This may be the reason. https://github.com/openelb/openelb/blob/master/deploy/porter.yaml#L1146-L1147

+1. This reminder should be noticed in the OpenELB documentation.

**Comment by quzhixue-Kimi** [comment]
kubectl apply -f https://raw.githubusercontent.com/kubesphere/porter/master/deploy/porter.yaml

It seems that https://raw.githubusercontent.com/kubesphere/porter/master/deploy/porter.yaml  404 not found.

Sorry for your inconvenience.
BR
Kimi

**Comment by FeynmanZhou** [comment]
> kubectl apply -f https://raw.githubusercontent.com/kubesphere/porter/master/deploy/porter.yaml
> 
> It seems that https://raw.githubusercontent.com/kubesphere/porter/master/deploy/porter.yaml 404 not found.
> 
> ![Screenshot (21)](https://user-images.githubusercontent.com/8235746/155685601-15d5a245-79e8-4a87-923c-574d90a358f3.png)
> 
> Sorry for your inconvenience. BR Kimi

The file name has been updated to `openelb.yaml`, see https://github.com/openelb/openelb/issues/241.

Hi @renyunkang , could you please update the OpenELB documentation accordingly?

**Comment by renyunkang** [comment]
> > kubectl apply -f https://raw.githubusercontent.com/kubesphere/porter/master/deploy/porter.yaml
> > It seems that https://raw.githubusercontent.com/kubesphere/porter/master/deploy/porter.yaml 404 not found.
> > ![Screenshot (21)](https://user-images.githubusercontent.com/8235746/155685601-15d5a245-79e8-4a87-923c-574d90a358f3.png)
> > Sorry for your inconvenience. BR Kimi
> 
> The file name has been updated to `openelb.yaml`, see #241.
> 
> Hi @renyunkang , could you please update the OpenELB documentation accordingly?

ok, i will update OpenELB English documentation as soon as possible.

**Comment by FeynmanZhou** [comment]
> kubectl apply -f https://raw.githubusercontent.com/kubesphere/porter/master/deploy/porter.yaml
> 
> It seems that https://raw.githubusercontent.com/kubesphere/porter/master/deploy/porter.yaml 404 not found.
> 
> ![Screenshot (21)](https://user-images.githubusercontent.com/8235746/155685601-15d5a245-79e8-4a87-923c-574d90a358f3.png)
> 
> Sorry for your inconvenience. BR Kimi

@quzhixue-Kimi ,

This issue has been fixed, you can follow the latest documents to install OpenELB. 

Thanks

**Comment by quzhixue-Kimi** [comment]
Hi there,

It is appreciated that you could provide me with the latest document on how to install the OpenELB.

Is that the latest one via https://openelb.github.io/docs/getting-started/installation/install-porter-on-kubernetes/?

![Screenshot (24)](https://user-images.githubusercontent.com/8235746/155949586-7baeeefe-3bff-4849-b37f-4139020ef04c.png)

It seems that the installation has not been updated.

BR
Kimi

**Comment by FeynmanZhou** [comment]
> https://openelb.github.io/docs/getting-started/installation/install-porter-on-kubernetes/

Please check out: https://openelb.github.io/docs/getting-started/installation/install-openelb-on-kubernetes/

It's too odd that the previous links are still available. We will investigate this problem.

**Comment by quzhixue-Kimi** [comment]
Hi there,

Thank you for your feedback.
But, maybe, you may update the https://raw.githubusercontent.com/openelb/openelb/master/deploy/openelb.yaml
image: kubesphere/openelb:v0.4.3

It seems that the latest tag is v0.4.4 and, v0.4.3 could not be found via https://hub.docker.com/r/kubesphere/openelb/tags

BR
Kimi

**Comment by FeynmanZhou** [comment]
> Hi there,
> 
> Thank you for your feedback. But, maybe, you may update the https://raw.githubusercontent.com/openelb/openelb/master/deploy/openelb.yaml image: kubesphere/openelb:v0.4.3
> 
> It seems that the latest tag is v0.4.4 and, v0.4.3 could not be found via https://hub.docker.com/r/kubesphere/openelb/tags
> 
> BR Kimi

Good catch. @renyunkang it looks like the image name is not up to date.
@quzhixue-Kimi Could you raise a GitHub PR to update it to v0.4.4?

**Comment by FeynmanZhou** [comment]
> Hi there,
> 
> Thank you for your feedback. But, maybe, you may update the https://raw.githubusercontent.com/openelb/openelb/master/deploy/openelb.yaml image: kubesphere/openelb:v0.4.3
> 
> It seems that the latest tag is v0.4.4 and, v0.4.3 could not be found via https://hub.docker.com/r/kubesphere/openelb/tags
> 
> BR Kimi

Yes, it looks like this PR will fix it: https://github.com/openelb/openelb/pull/247/files
